README.md

```diff
@@ -56,45 +56,56 @@ This library currently supports:
 
 ## Installation
 
-This library is available through [npm](https://www.npmjs.com/package/@okta/okta-react). To install it, simply add it to your project:
+This library is available through [npm](https://www.npmjs.com/package/@okta/okta-react).
+
+Install `@okta/okta-react`
 
 ```bash
 npm install --save @okta/okta-react
 ```
 
+Install peer dependency `@okta/okta-auth-js`
```

Comment:
Note - npm 7 installs peer dependencies by default.  I don't think that's a problem for us, but please take a moment to consider if I'm wrong.

Reply:
I think a manual install will not hurt even if it's installed by default in npm7, as it can also handle cases for package managers < npm7 or yarn.